src/stories/MoleculeFollowerGoal.stories.tsx

```diff
@@ -1,20 +1,23 @@
 import { ComponentProps } from 'react';
 // also exported from '@storybook/react' if you can deal with breaking changes in 6.1
 import { Story, Meta } from '@storybook/react/types-6-0';
-
-import { MoleculeFollowerGoalStory } from './MoleculeFollowerGoal';
+import mock, { followers, users } from './mock';
+import { TWITCH_API_ENDPOINT } from '../constants';
+import { FollowerGoal } from '../components/molecules';
 
 export default {
   title: 'Molecule/FollowerGoal',
-  component: MoleculeFollowerGoalStory,
-  argTypes: {
-    backgroundColor: { control: 'color' },
-  },
+  component: FollowerGoal,
+  argTypes: {},
 } as Meta;
 
-const Template: Story<
-  ComponentProps<typeof MoleculeFollowerGoalStory>
-> = args => <MoleculeFollowerGoalStory {...args} />;
+const Template: Story<ComponentProps<typeof FollowerGoal>> = args => {
+  mock.restore();
+  mock.mock(`${TWITCH_API_ENDPOINT}users`, users.me);
+  mock.mock(`begin:${TWITCH_API_ENDPOINT}users/follows`, followers.me);
```

Comment:
I think we can just move all the mocking setup in `preview.tsx`, so it doesn't have to be redone in all individual stories.

Reply:
Done